.rubocop.yml

```diff
@@ -5,7 +5,7 @@ require:
 inherit_from: .rubocop_todo.yml
 
 AllCops:
-  TargetRubyVersion: 3.0
+  TargetRubyVersion: 3.1
```

Comment:
A wee bump as we march towards Ruby 3.1. Nice to reduce all the duplication in pre-3.1 hash syntax.